src/components/ResultListItem.tsx

```diff
@@ -104,6 +104,10 @@ const StyledHeader = styled.div`
 `;
 
 const StyledChip: any = styled(Chip)`
+  max-width: 13rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.lg - 1 + 'px'}) {
```

Comment:
er den -1 for å være sikker, eller er det noe som hakker om ikke ?

Reply:
Fjernet bredden og lagt til multiline istedet